recipes/default.rb

```diff
@@ -64,6 +64,17 @@
   not_if "mysql -h #{node[:drupal][:db][:host]} -u root -p#{node[:mysql][:server_root_password]} --silent --skip-column-names --execute=\"show databases like '#{node[:drupal][:db][:database]}'\" | grep #{node[:drupal][:db][:database]}"
 end
 
+template "#{node['drupal']['dir']}/sites/default/settings.php" do
+  source "d7.settings.php.erb"
+  mode "0644"
+  variables(
+    'database'        => node['drupal']['db']['database'],
```

Comment:
No big deal but the formatting here is quite different. You can use strings but a ruby token is a lot more... rubyish. so just `:database` and `:user` etc for these. Also the space formatting on the rest are just `:database => node[:drupal][:db][:database]`